provider/resources.go

```diff
@@ -100,6 +100,7 @@ func Provider() tfbridge.ProviderInfo {
 			"fastly_service_acl_entries_v1":             {Tok: makeResource(mainMod, "ServiceACLEntriesv1")},
 			"fastly_service_dictionary_items_v1":        {Tok: makeResource(mainMod, "ServiceDictionaryItemsv1")},
 			"fastly_service_dynamic_snippet_content_v1": {Tok: makeResource(mainMod, "ServiceDynamicSnippetContentv1")},
+			"fastly_user_v1":                            {Tok: makeResource(mainMod, "UserV1")},
```

Comment:
Is it important that all other `v`'s in `v1` are lowercase while this one is `V1` upper?

Reply:
Fixed!